test/tools/integration/run_e2e_tests.sh

```diff
@@ -13,6 +13,9 @@ export AWS_E2E_TESTS_KEY_ID=$AWS_E2E_TESTS_KEY_ID
 export AWS_E2E_TESTS_SECRET=$AWS_E2E_TESTS_SECRET
 export DO_E2E_TESTS_TOKEN=$DO_E2E_TESTS_TOKEN
 export HZ_E2E_TOKEN=$HZ_E2E_TOKEN
+export VSPHERE_E2E_PASSWORD=$VSPHERE_PASSWORD
+export VSPHERE_E2E_USERNAME=$VSPHERE_USERNAME
+export VSPHERE_E2E_ADDRESS=$VSPHERE_ADDRESS
```

Comment:
Do I have to generate the new username and password ? Or Can I use the ones you gave me ?